docs/Machine Learning/linear_regression.md

```diff
@@ -0,0 +1,98 @@
+# Linear Regression
+
+### Introduction to Linear Regression
+
+Linear regression is a fundamental and widely used statistical technique for predicting numeric outcomes based on one or more independent variables. This document provides an overview of linear regression, including its principles, advantages, disadvantages, and practical considerations for application.
+
+### Overview of Linear Regression Model
+
+Linear regression models the relationship between a dependent variable (target) and one or more independent variables (predictors) by fitting a linear equation to observed data.
+
+### Formula:
+
+#### Y = 𝛽0 + 𝛽1x1 +  𝛽2x2 + 𝛽3x3..........𝛽nxn + 𝜖

```

Comment:
replace `Y = 𝛽0 + 𝛽1x1 + 𝛽2x2 + 𝛽3x3..........𝛽nxn + 𝜖` to `$\ Y = \beta_0 + \beta_1 x_1 + \beta_2 x_2 + \beta_3 x_3 + \cdots + \beta_n x_n + \epsilon \$`